ISX/ISX-CabinCrew/CabinCrewViewController.swift

```diff
@@ -134,14 +133,17 @@ class CabinCrewViewController: UITableViewController {
      Returns array with products
     */
     func observeProducts() {
+        productsArray = []
         productsRef?.observe(.value, with: { snapshot in
             for item in snapshot.children {
                 if let productSnapshot = item as? DataSnapshot {
                     let product = Product(snapshot: productSnapshot)
                     self.productsArray.append(product)
                 }
             }
+            self.activityIndicatorView?.stopAnimating()
             self.tableView.reloadData()
+            self.initialLoad = true
```

Comment:
Because it's the first time the page was loaded, after that, we can reload the page after someone revisits the page. and without it, the observers would have been run twice on first load